src/ParallelDataCollector.cpp

```diff
@@ -1139,4 +1112,56 @@ namespace splash
         dataset.close();
     }
 
+    /* UNIMPLEMENTED METHODS FROM DATACOLLECTOR. TODO: Unify interface to remove those */
+    void ParallelDataCollector::readGlobalAttribute(const char*, void*, Dimensions*)
```

Comment:
I don't get those: PDC *should* already have a usable `readGlobalAttribute` method, no?